app/cli.php

```diff
@@ -3,30 +3,78 @@
 require_once __DIR__ . '/init.php';
 require_once __DIR__ . '/controllers/general.php';
 
-use Utopia\App;
+use Appwrite\CLI\Tasks;
 use Utopia\CLI\CLI;
-use Utopia\CLI\Console;
 use Utopia\Database\Validator\Authorization;
+use Utopia\Platform\Service;
+use Utopia\App;
+use Utopia\CLI\Console;
+use Utopia\Cache\Adapter\Redis as RedisCache;
+use Utopia\Cache\Cache;
+use Utopia\Database\Adapter\MariaDB;
+use Utopia\Database\Database;
 
 Authorization::disable();
 
-$cli = new CLI();
-
-include 'tasks/doctor.php';
-include 'tasks/maintenance.php';
-include 'tasks/install.php';
-include 'tasks/migrate.php';
-include 'tasks/sdks.php';
-include 'tasks/specs.php';
-include 'tasks/ssl.php';
-include 'tasks/vars.php';
-include 'tasks/usage.php';
-
-$cli
-    ->task('version')
-    ->desc('Get the server version')
-    ->action(function () {
-        Console::log(App::getEnv('_APP_VERSION', 'UNKNOWN'));
-    });
+CLI::setResource('register', fn()=>$register);
+
+$attempts = 0;
+$max = 10;
+$sleep = 1;
+
+do {
```

Comment:
We should avoid having logic in the global scope, this is one of the DI's main benefits.

Reply:
updated.